content/docs/rfcs/14/README.md

```diff
@@ -1,120 +1,102 @@
+# Abstract
+
+Waku v2 is a family of modular peer-to-peer protocols for secure communication. These protocols are designed to be secure, privacy-preserving, censorship-resistant, and able to run in resource-restricted environments. At a high level, Waku v2 implements a Pub/Sub messaging pattern over libp2p and adds capabilities to it.
+
+The present document specifies the Waku v2 message format and a way to encapsulate the messages sent with specific information security goals.
 
 # Motivation
 
-When sending messages over Waku there are multiple concerns:
+When sending messages over Waku, there are multiple concerns:
+
 - We may have a separate encryption layer as part of our application
-- We may want to provide efficient routing for resource restricted devices
+- We may want to provide efficient routing for resource-restricted devices
 - We may want to provide compatibility with Waku v1 envelopes
 - We may want payloads to be encrypted by default
 - We may want to provide unlinkability for metadata protection
 
 This specification attempts to provide for these various requirements.
 
-# WakuMessage
+# Semantics
+
+## Waku Message
 
-A `WakuMessage` is what is being passed around by the other protocols, such as WakuRelay, WakuStore, and WakuFilter.
+A waku message is constituted by the combination of data and attributes that, for example, a *publisher* sends to a *topic* and is eventually delivered to *subscribers*.
 
-The `payload` field SHOULD contain whatever payload is being sent. See section below on payload encryption.
+## Message Attributes
 
-The `contentTopic` field SHOULD be filled out to allow for content-based filtering.
-See [12/WAKU2-FILTER](/spec/12) and [13/WAKU2-STORE](/spec/13) for more details.
-To enable a bidirectional bridge with Waku v1 see [15/WAKU2-BRIDGE](/spec/15) for further requirements on this field.
 
-The `version` field MAY be filled out to allow for various types of payload encryption.
-Omitting it means the version is 0.
+The `payload` attribute SHOULD contain the data to be sent.
 
-The `timestamp` field MAY be filled out to signify the time at which the message is generated by its sender. 
-This field holds the Unix epoch time in nanoseconds. 
-Omitting it means the timestamp is unspecified.
+The `contentTopic` attribute SHOULD specify a string identifier that can be used for content-based filtering.
 
-The `ephemeral` field MAY be set to signify the transient nature of the message.
-If the message should be stored by the [store protocol](/spec/13), then this field MUST be set to `false`, which is equivalent to omitting the field.
-If the message should not be stored by the [store protocol](/spec/13), then this field MUST be set to `true`.
+The `version` attribute MAY contain a version number to discriminate different types of payload encryption. If omitted, the value SHOULD be interpreted as version 0.
```

Comment:
How about semantic line breaks?

Reply:
What do you mean by "semantic line breaks"? 👀 

I see two related and complementary phrases that do not require any line break for clarity/better readability.

Reply:
I usually separate sentences after the period `.`, but not a big deal.